### DISCUSSION TITLE
Funcionarios

### DISCUSSION THREAD

**Discussion by Felipebzzz** [RamonSilva20/mapos#1669] [General]
Poderia fazer uma parte Funcionarios. E nada parte do Mergado pago na OS, ele da erro

**Comment by willph** [comment]
MercadoPago tem bug. 

Gerencianet e asaas é os que funcionam.

Poderia fazer uma parte funcionários?

**Reply by Felipebzzz** [reply]
Pois assim, usei o modelo de voces ,e to fazendo um erp pra empresa, e to fazendo a parte de funcionarios, rh, compras, essas coisas. Ai na parte do mercado pago, da bugs, pensei que fosse a aplicação que eu tivesse feito algo errado

**Comment by willph** [comment]
E da API do mercado pago mesmo.

Em modo sandbox funciona tranquilo. 
Mas em modo produção ele gera o boleto, porém as outras funcionalidades não pega.

**Reply by Felipebzzz** [reply]
Assim, pra mim essa funcionalidade nao irei usar na aplicação que estou fazendo. Retirei a API. Muito obrigado

**Comment by Felipebzzz** [comment]
Uma pergunta, toda vez que eu cadastro ,ou faço algo no sistema, o alerta fica na tela e depois de 5 seg apaga, o que poderia ser?

**Comment by Pr3d4dor** [comment]
@Felipeb96 Qual versão do mapos?

**Reply by Felipebzzz** [reply]
4.24.0

**Comment by Pr3d4dor** [comment]
Tenta atualizar para a última versão.

**Comment by Felipebzzz** [comment]
Mas eu ja fiz varias modificações nele, e ja deixei totalmente diferente . Só falta eu arrumar este bug

**Comment by Pr3d4dor** [comment]
@Felipeb96 Infelizmente como é uma versão modificada, fica difícil dar suporte. Apenas ajudamos com o mapos original.

**Reply by Felipebzzz** [reply]
Boa noite, na parte de financeiro, o modo editar, nao puxa alguns campos do banco de dados, alguma sugestão?!
